cluster/log-dump/log-dump.sh

```diff
@@ -218,6 +218,25 @@ function export-windows-docker-event-log() {
     done
 }
 
+# Saves prepulled Windows Docker images list to ${WINDOWS_LOGS_DIR}\docker_images.log
+# on node $1.
+function export-windows-docker-images-list() {
+    local -r node="${1}"
+
+    local -r powershell_cmd="powershell.exe -Command \"\$logs=\$(docker image list); \$logs | Out-File -FilePath '${WINDOWS_LOGS_DIR}\\docker_images.log'\""
+
+    # Retry up to 3 times to allow ssh keys to be properly propagated and
+    # stored.
+    for retry in {1..3}; do
+      if gcloud compute ssh --project "${PROJECT}" --zone "${ZONE}" "${node}" \
+        --command "$powershell_cmd"; then
```

Comment:
```suggestion
        --command "${powershell_cmd}"; then
```
Use braced variable expansions.

https://google.github.io/styleguide/shellguide.html#s5.6-variable-expansion